--- origin_omp/Makefile	2020-08-23 03:12:30.429735711 -0400
+++ bridges.omp.Makefile	2020-08-23 03:24:11.000000000 -0400
@@ -48,10 +48,10 @@ serialCXX    = icc
 # Compiler to use with MPI parallelism
 parallelCXX  = mpiicc
 # General compiler flags (e.g. -Wall to turn on all warnings on g++)
-compileFlags = -Wall -Wextra -Wnon-virtual-dtor -Wno-deprecated-declarations -DPLB_MAC_OS_X -std=gnu99 -D_OPENMP -qopenmp
+compileFlags = -Wall -Wextra -Wnon-virtual-dtor -Wno-deprecated-declarations -DPLB_MAC_OS_X -std=gnu99 -D_OPENMP -qopenmp -xCORE-AVX2
 
 # General linker flags (don't put library includes into this flag)
-linkFlags    = -lm -qopenmp
+linkFlags    = -lm -qopenmp -xCORE-AVX2
 # Compiler flags to use when optimization mode is on
 optimFlags   = -O3
 #optimFlags   = -xHOST -O3 -ip -no-prec-div -static
