--- origin/Makefile	2020-08-18 17:35:12.000000000 -0500
+++ Makefile.seq.stampede2	2020-08-18 17:46:33.000000000 -0500
@@ -38,17 +38,17 @@ usePOSIX     = true
 srcPaths =
 # Path to external libraries (other than Palabos)
 libraryPaths =
-# Path to inlude directories (other than Palabos)
-includePaths = -I/opt/intel/vtune_amplifier_2019.6.0.602217/include
+# Path to external libraries (other than Palabos)
+includePaths = -I/opt/intel/vtune_amplifier_2018.4.0.574913/include
 # Dynamic and static libraries (other than Palabos)
-libraries    = /opt/intel/vtune_amplifier_2019.6.0.602217/lib64/libittnotify.a
+libraries    = /opt/intel/vtune_amplifier_2018.4.0.574913/lib64/libittnotify.a
 
 # Compiler to use without MPI parallelism
 serialCXX    = icc
 # Compiler to use with MPI parallelism
 parallelCXX  = mpiicc
 # General compiler flags (e.g. -Wall to turn on all warnings on g++)
-compileFlags = -Wall -Wextra -Wnon-virtual-dtor -Wno-deprecated-declarations -DPLB_MAC_OS_X -std=gnu99
+compileFlags = -Wall -Wextra -Wnon-virtual-dtor -Wno-deprecated-declarations -DPLB_MAC_OS_X -std=gnu99 ${TACC_VEC_FLAGS}
 
 # General linker flags (don't put library includes into this flag)
 linkFlags    = -lm
